modelmapper/cleaner.py

```diff
@@ -162,34 +192,39 @@ def xls_date(x):
                 field_values[:] = map(lambda x: None if x is None else strptime(x, _format), field_values)
         return field_values
 
-    def clean(self, content_type, path=None, content=None, sheet_names=None):
+    def clean(self, content_type, path=None, content=None, sheet_names=None, ignore_new_fields=True):
         """
         Clean the data for importing into database.
         content_type: Options: csv, xls, xls_xml, xlsx
         path: (optional) The path to the file to open
         content: (optional) The content to be read. The content can be bytes, string, BytesIO or StringIO
         sheet_names: (optional) The sheet names from the Excel file to be considered.
                                 If none provided, all sheets will be considered.
+        ignore_new_fields: (optional) If true: fields not defined in the model will be dropped from the data source
```

Comment:
`ignore_missing_fields: (optional) If true: fields not found in the model will be ignored`